contracts/DIDToken.sol

```diff
@@ -1,23 +1,24 @@
 pragma solidity ^0.4.11;
 
-import './lib/SafeMath.sol';
 import './lib/Approvable.sol';
+import './lib/SafeMath.sol';
 import './lib/Token.sol';
 
 contract DIDToken is Approvable, Token {
   using SafeMath for uint256;
 
-  event LogReward(address indexed to, uint256 numDID);
+  event LogIssueDID(address indexed to, uint256 numDID);
 
   function DIDToken () {
-    name = "Distense DID";
-    symbol = "DID";
+    name = 'Distense DID';
+    symbol = 'DID';
   }
 
-  function issueReward(address _recipient, uint256 _amount) external onlyApproved returns (bool) {
-    totalSupply = totalSupply.add(_amount);
-    balances[_recipient] = balances[_recipient].add(_amount);
-    LogReward(_recipient, _amount);
+  function issueDID(address _recipient, uint256 _numDID) external onlyApproved returns (bool) {
+    require(_recipient != address(0));
```

Comment:
That's kind of what I was wondering about how and where to validate addresses.  Seems like they should sort of be everywhere, especially when sending money or when the specific address matters for authentication.

"all symbols visible from the function are visible in the modifier" so we can just do:

`function issueDID(address _recipient, uint256 _numDID) external validAddress onlyApproved`